src/Microsoft.Azure.ServiceBus/Amqp/AmqpMessageConverter.cs

```diff
@@ -562,6 +647,17 @@ static AmqpMap GetSqlFilterMap(SqlFilter sqlFilter)
             return amqpFilterMap;
         }
 
+        static SqlFilter GetSqlFilter(AmqpMap map)
+        {
+            string sqlExpression;
+            if (map.TryGetValue(ManagementConstants.Properties.Expression, out sqlExpression))
```

Comment:
Removed this method

---
In reply to: [128698425](https://github.com/Azure/azure-service-bus-dotnet/pull/192#discussion_r128698425) [](ancestors = 128698425)